mapie/estimators.py

```diff
@@ -165,10 +165,16 @@ def _check_parameters(self) -> None:
             raise ValueError("Invalid ensemble argument. Must be a boolean.")
 
         if not isinstance(self.n_jobs, (int, type(None))):
-            raise ValueError("Invalid n_jobs argument. Must be a integer.")
+            raise ValueError("Invalid n_jobs argument. Must be an integer.")
+
+        if self.n_jobs == 0:
+            raise ValueError("Invalid n_jobs argument. Must be different than 0.")
```

Comment:
En toute rigueur, -2, -3 etc ne sont pas acceptés non plus

Reply:
Les valeurs négatives sont acceptées par joblib: "For n_jobs below -1, (n_cpus + 1 + n_jobs) are used."